### PR TITLE
2170 - update dependencies

### PR DESCRIPTION
**Pull request by cogross**
Updated dependencies listed in the gitlab dependabot that are findings.

### CODE REVIEW

**Review by avgAGB — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -54,19 +54,21 @@
...
         <version.easymock>4.0.2</version.easymock>
         <version.eclipse.emf>2.15.0</version.eclipse.emf>
-        <version.geoserver>2.22.1</version.geoserver>
+        <version.geoserver>2.22.5</version.geoserver>
         <version.geotools>28.1</version.geotools>
         <version.geowave>1.2.0</version.geowave>
         <version.google-guava>31.1-jre</version.google-guava>
         <version.googlecode-findbugs>2.0.3</version.googlecode-findbugs>
         <version.googlecode-json-simple>1.1.1</version.googlecode-json-simple>
+        <version.guava>32.0.0-android</version.guava>
```
Comment (ivakegg) [inline pom.xml]: Where is this being used?

File: warehouse/pom.xml [hunk 2]
```diff
@@ -357,7 +357,7 @@
             <dependency>
                 <groupId>org.json</groupId>
                 <artifactId>json</artifactId>
-                <version>20080701</version>
+                <version>20231013</version>
```
Comment (ivakegg) [inline warehouse/pom.xml]: Lets create a version property for this one

File: warehouse/pom.xml [hunk 3]
```diff
@@ -372,7 +372,7 @@
             <dependency>
                 <groupId>org.python</groupId>
                 <artifactId>jython</artifactId>
-                <version>2.5.2b2-callable</version>
+                <version>2.7.2b2</version>
```
Comment (ivakegg) [inline warehouse/pom.xml]: create a version property

File: warehouse/pom.xml (hunk 2 above)
Comment (cogross) [reply warehouse/pom.xml]: sure

File: warehouse/pom.xml (hunk 3 above)
Comment (cogross) [reply warehouse/pom.xml]: sure

File: pom.xml (hunk 1 above)
Comment (cogross) [reply pom.xml]: nowhere!  pulled it, and everything builds and tests run.

**Review by apmoriarty — APPROVED** [review]

**Review by avgAGB — APPROVED** [review]

**Pull request merged**